app/scripts/modules/openstack/serverGroup/details/serverGroupDetails.openstack.controller.js

```diff
@@ -346,5 +348,13 @@ module.exports = angular.module('spinnaker.serverGroup.details.openstack.control
       });
     };
 
+    this.applyFloatingIpDetails = () => {
+      return networkReader.listNetworksByProvider('openstack').then((networks) => {
+        ctrl.floatingNetworkName = (_.find(networks, (net) => {
```

Comment:
This block would read a little better if it used either all `ctrl`s or all `this`s